app/assets/javascripts/discourse/app/templates/review-index.hbs

```diff
@@ -1,3 +1,27 @@
+{{#if this.displayUnknownReviewableTypesWarning}}
+  <div class="alert alert-info unknown-reviewables">
+    <span class="text">{{i18n "review.unknown.title"}}</span>
+    <ul>
+      {{#each this.unknownReviewableTypes as |type|}}
+        <li>{{type}}</li>
+      {{/each}}
+    </ul>
+    <span class="text">{{htmlSafe
+        (i18n
+          "review.unknown.instruction"
+          url="https://meta.discourse.org/t/types-of-review-items-created-by-plugins/350179"
```

Comment:
```suggestion
          url="https://meta.discourse.org/t/350179"
```

Minor, but we should always try use this form (or `https://meta.discourse.org/t/-/350179`), it's more succinct and doesn't get outdated if the topic title changes